binscripts/rvm-installer

```diff
@@ -219,7 +218,7 @@ __rvm_curl()
 )
 
 rvm_error()  { printf "ERROR: %b\n" "$*"; }
-__rvm_which(){   which "$@" || return $?; true; }
+__rvm_which(){ command -v "$@" 2>/dev/null; }
```

Comment:
@havenwood All of the files in this PR have the shebang `#!/usr/bin/env bash`, so zsh compatibility should not be a requirement. If it is, then the shebang should probably read `!#/bin/sh`.